client/src/App.js

```diff
@@ -13,16 +13,28 @@ export default class App extends Component {
   }
 
   addToSavedList = movie => {
+    let newItem = true;
     const savedList = this.state.savedList;
+    savedList.map(item => {
+      if (movie.title === item.title) {
+        newItem = false;
+      }
+    })
+    if(newItem) {
     savedList.push(movie);
     this.setState({ savedList });
+  }
   };
 
   render() {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+      <Route path='/' exact
+             component={() => <MovieList onClick={this.addToSavedList}/>}>
```

Comment:
Even though this works, its not the way that you want to do it. Instead of `component` you should you `render props`